exe-common/src/network/peer.rs

```diff
@@ -29,6 +31,13 @@ impl Peer {
             config::net::Peer::Http(addr) => {
                 Ok(Peer::Http(hermes::HermesEndPoint::new(addr, network)))
             }
+            config::net::Peer::Ntt(addr) => {
+                let mut addrs_iter = addr.to_socket_addrs().unwrap();
```

Comment:
The `unwrap` is still there, meaning that jormungandr will panic on any invalid address?

Reply:
I hope this code is never used in jormundandr.. but you are right that cardano-cli will fail

Reply:
Sorry, I meant cardano-cli. It should be easy to check for the error here and convert it to the same `InvalidPeerAddress` error variant.

Reply:
Fixed

Reply:
Hmm, the `unwrap` is still there?